src/Stratis.StratisD/Program.cs

```diff
@@ -31,15 +33,22 @@ public static void Main(string[] args)
                 NodeSettings nodeSettings = new NodeSettings(network, ProtocolVersion.ALT_PROTOCOL_VERSION).LoadArguments(args);
 
                 // NOTES: running BTC and STRAT side by side is not possible yet as the flags for serialization are static
-
                 var node = new FullNodeBuilder()
                     .UseNodeSettings(nodeSettings)
                     .UseStratisConsensus()
                     .UseBlockStore()
                     .UseMempool()
                     .UseWallet()
                     .AddPowPosMining()
-                    .UseApi()
+                    .UseApi(null, options =>
+                    {
```

Comment:
Can we not move this in to the feature? would make the builder part much cleaner.

Reply:
no you can't that's how this is designed. You get an options object, that you need to initialize here.

Reply:
Fair enough. 
Optional we could add a second method in the feature to do that.
```
 .UseKeepAliveApi(args)
```

Reply:
Don't really like passing the whole args objects to features.
Also methods that look like  .UseXXX() are really used for features and I think the keepalive should stay as an api option as 1. it depends on it 2. it really means that if the api hasn't been called, kill the app.